target/mips/cp0_helper.c

```diff
@@ -719,6 +719,25 @@ void helper_dmtc0_entrylo0(CPUMIPSState *env, uint64_t arg1)
 }
 #endif
 
+#ifdef TARGET_CHERI
+
+void helper_mtc0_gc_lo(CPUMIPSState *env, target_ulong arg1)
```

Comment:
I recognize this is extremely minor, but I don't like the "gc" in here.  This isn't GC in the usual sense.  `capfilter`?

Reply:
Done